cmd/sendevent/sendevent.go

```diff
@@ -119,3 +138,24 @@ func (o *CliOptions) send(eventType, target, data string) error {
 	fmt.Printf("\nResponse: %s\n", response)
 	return nil
 }
+
+func readEventsFromFile(file string) ([]string, error) {
+	var rawEvents []json.RawMessage
+
+	fileData, err := ioutil.ReadFile(file)
```

Comment:
[`ioutil.ReadFile`](https://cs.opensource.google/go/go/+/refs/tags/go1.20.5:src/io/ioutil/ioutil.go;l=35) is deprecated, please switch to `os.ReadFile`:
```suggestion
	fileData, err := os.ReadFile(file)
```

Reply:
Resolved